docs/contents/user-docs/install.md

```diff
@@ -143,27 +85,20 @@ $ helm list --namespace $ACK_K8S_NAMESPACE -o yaml
   updated: 2020-12-17 13:09:17.309002201 -0500 EST
 ```
 
-## Static Kubernetes manifests
+## Install an ACK service controller with static Kubernetes manifests
 
-If you prefer not to use Helm, you may install a service controller using
-static Kubernetes manifests that are included in the source repository for an
-ACK service controller.
+If you prefer not to use Helm, you may install an ACK service controller using static Kubernetes manifests that are included in the source repository. 
 
-Static Kubernetes manifests that install the individual service controller as a
-Kubernetes `Deployment`, along with the relevant Kubernetes RBAC resources are
-available in the `config/` directory of the associated ACK service controller's
-source repository.
+Static Kubernetes manifests install an individual service controller as a Kubernetes deployment, including the relevant Kubernetes RBAC resources. Static Kubernetes manifests are available in the `config/` directory of the associated ACK service controller's source repository.
 
-For example, for the static manifests that will install the RDS service
-controller for ACK, check out the [`config/`][rds-config-dir] directory of the
-[RDS controller's source repo][rds-repo].
+For example, the static manifests needed to install the S3 service controller for ACK are available in the [`config/`][s3-config-dir] and [`helm/`][s3-helm-dir] directories in the [S3 controller's source repository][s3-repo].
```

Comment:
nit: Remove reference to `helm/`. Not relevant for the purposes of this section, and we don't recommend installing Helm using this directory.